graphene_django/decorators.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+from functools import wraps
+from django.http import HttpResponseForbidden
+
+
+def has_perms(permissions):
+    """
+    Check if have user logged and permissions to see some value
+
+    Example:
+        class CityNode(DjangoObjectType):
+            class Meta(object):
+                interfaces = (relay.Node,)
+                model = City
+                only_fields = (
+                    'name', 'locality', 'slug', 'state', 'active',
+                )
+
+            @has_perms(["django_city_app.can_see_location"])
+            def resolve_location(self, args, context, info):
+                return self.locality.pos
+
+    Args:
+        permissions: ["django_app.permission_codename",]
+    """
+    def decorator(method):
+        if callable(permissions):
+            method.permissions = []
+        else:
+            method.permissions = permissions
+
+        @wraps(method)
+        def wrapper(*args, **kwargs):
+            context = kwargs.get('context', dict(zip(method.func_code.co_varnames,
+                                                     args)).get('context', None))
+            if not context:
+                return HttpResponseForbidden('Forbidden. No context, no access.')
```

Comment:
I think is better to use the base `Exception` or custom created ones for this case.

### Why?
The `HTTPResponse*` exception implies only one triggered per request, which is not the case as could be triggered multiple times (one per each field resolution).